src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -5419,9 +5426,9 @@ private boolean classesFailingToCompile() {
     }
 
     // Returns true if features have been modified in the configuration directory
+    // (excluding generated-features.xml file). Called only if generate-features=true
```

Comment:
Curious about this comment. It seems like `serverFeaturesModified()` is called on line 4030 regardless of the `generateFeatures` flag value. It also seems like the return value from this method is intended to determine whether or not we should install features regardless of `generateFeatures` value. Is this the intention?

Reply:
You're correct it is no longer used only in the geenrateFeatures scenario, now we use it outside of that scenario to avoid creating and copying to a temp dir if the server features have not changed. I've updated the comment